content/en/continuous_integration/pipelines/circleci.md

```diff
@@ -44,6 +44,31 @@ The Datadog integration for [CircleCI][1] works by using [webhooks][2] to send d
 
 2. Click **Add Webhook** to save the new webhook.
 
+### Configure multiple projects in bulk
+
+If you want to enable the hooks for many or all your CircleCI projects, Datadog provides a [script](https://raw.githubusercontent.com/DataDog/ci-visibility-circle-ci/main/service_hooks.py) to help you do it through the Circle CI API.
+
+To run the script, you need:
+
+- A Circle CI personal API token
+- To log in to your Circle CI account and follow all the projects for which you want to enable the hooks. You may use the "Follow All" button located at the top of the Projects page.
+
+The script only needs python3 and the requests package. For more info, run:
+```shell
+./service_hooks.py --help
+```
+
+The script supports environment variables `DD_API_KEY` and `DD_SITE`, and flag parameters `--dd-api-key` and `--dd-site`.
+
+Example:
+
+```
+./service_hooks.py \
```

Comment:
you could add `--dd-site {{< region-param key="dd_site" code="true" >}}` this would automatically add the currently selectd site in the docs website to the example

Reply:
Nice, I didn't know that.